src/System.Reflection.TypeExtensions/tests/Type/TypeInfoAPIs.cs

```diff
@@ -1,241 +0,0 @@
-        {
-            VerifyResults("DeclaredEvents", s_runtimeType.GetEvents(s_declaredOnly), s_typeInfo.DeclaredEvents);
-        }
-
-        [Fact]
-        public void Test11()
-        {
-            VerifyResults("DeclaredConstructors", s_runtimeType.GetConstructors(s_declaredOnly), s_typeInfo.DeclaredConstructors);
-        }
-
-        [Fact]
-        public void Test12()
-        {
-            VerifyResults("GetEvents", s_runtimeType.GetEvents(), s_typeInfo.AsType().GetEvents());
-        }
-
-        [Fact]
-        public void Test13()
-        {
-            VerifyResults("GetFields", s_runtimeType.GetFields(), s_typeInfo.AsType().GetFields());
-        }
-
-        [Fact]
-        public void Test14()
-        {
-            VerifyResults("GetMethods", s_runtimeType.GetMethods(), s_typeInfo.AsType().GetMethods());
-        }
-        [Fact]
-        public void Test15()
-        {
-            BindingFlags all = BindingFlags.DeclaredOnly | BindingFlags.FlattenHierarchy | BindingFlags.IgnoreCase | BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static;
-            VerifyResults("GetNestedTypes", s_runtimeType.GetNestedTypes(all), s_typeInfo.AsType().GetNestedTypes(all));
-        }
-
-        [Fact]
-        public void Test16()
-        {
-            VerifyResults("GetProperties", s_runtimeType.GetProperties(), s_typeInfo.AsType().GetProperties());
-        }
-
-        [Fact]
-        public void Test17()
-        {
-            VerifyResults("GetType", s_runtimeType.GetType(), s_typeInfo.GetType());
-        }
-
-        [Fact]
-        public void Test18()
-        {
-            VerifyResults("IsAssignableFrom", true, s_runtimeType.IsAssignableFrom(s_typeInfo.AsType()));
-        }
-
-        [Fact]
-        public void Test19()
-        {
-            VerifyResults("Equals", true, s_runtimeType.Equals(s_typeInfo));
-        }
-
-        [Fact]
-        public void Test20()
-        {
-            VerifyResults("IsClass", s_runtimeType.GetTypeInfo().IsClass, s_typeInfo.IsClass);
-        }
-
-        [Fact]
-        public void Test21()
-        {
-            VerifyResults("IsPublic", s_runtimeType.GetTypeInfo().IsPublic, s_typeInfo.IsPublic);
-        }
-
-        [Fact]
-        public void Test22()
-        {
-            VerifyResults("IsGenericType", s_runtimeType.GetTypeInfo().IsGenericType, s_typeInfo.IsGenericType);
-        }
-
-        [Fact]
-        public void Test23()
-        {
-            VerifyResults("IsImport", s_runtimeType.GetTypeInfo().IsImport, s_typeInfo.IsImport);
-        }
-
-        [Fact]
-        public void Test24()
-        {
-            VerifyResults("IsEnum", s_runtimeType.GetTypeInfo().IsEnum, s_typeInfo.IsEnum);
-        }
-
-        [Fact]
-        public void Test25()
-        {
-            VerifyResults("IsGenericTypeDefinition ", s_runtimeType.GetTypeInfo().IsGenericTypeDefinition, s_typeInfo.IsGenericTypeDefinition);
-        }
```

Comment:
Where did all of these "Is" tests go, e.g. IsClass, IsPublic, etc.?  Are they already covered elsewhere?